### DISCUSSION TITLE
protoc import "cue/cue.proto"; ?

### DISCUSSION THREAD

**Discussion by randomizedcoder** [cuelang/cue#983] [General]
G'day,

I'm a newbie attempting to use Cue for the first time, so sorry for the really basic questions. I figure I should ask to help the next newbie, and possibly add a little to the docs to make it more clear. - Documentation does seems great BTW

Essentially, I'm hoping to this CUE mostly to code gen input validation for gRPC, per the tail end of:
https://cuelang.org/docs/usecases/generate/

I've added the cue tags to the .protos, per:
https://cuelang.org/docs/integrations/protobuf/#extract-cue-from-a-standalone-proto-file

I've done the install via: https://cuelang.org/docs/install/
Added the import for cue, just after the standard timestamp, for example:

```
import "google/protobuf/timestamp.proto";
import "cue/cue.proto";
```

Then I tried protoc, but I suspect I'm missing a plugin.

```
/bin/protoc *.proto --go_out=plugins=grpc:/my/go/src/
/bin/protoc *.proto --go_out=plugins=grpc:/home/das/go/src/
cue/cue.proto: File not found.
mp.proto: Import "cue/cue.proto" was not found or had errors.
```

But I don't know what to do.  Searching finds the envoyproxy stuff, and actually that might be an alternative to cue.  I don't know what the community is doing, with CUE verse this envoy proxy code gen thinger.

https://github.com/envoyproxy/protoc-gen-validate

Thanks in advance

**Comment by cueckoo** [comment]
This discussion has been migrated to https://github.com/cue-lang/cue/issues/983.

For more details about CUE's migration to a new home, please see https://github.com/cue-lang/cue/issues/1078.
